kod/object/item/passitem/numbitem/ammo/firearow.kod

```diff
@@ -21,8 +18,11 @@ resources:
 
    fire_arrow_name_rsc = "fire arrows"
    fire_arrow_icon_rsc = arrowfir.bgf
-   fire_arrow_desc_rsc = "These arrows do excellent damage, burning "
-      " brightly, they burn for extra damage."
+   fire_arrow_desc_rsc = \
+      "These arrows do excellent damage, burning "
+      " brightly, scorching your enemy."
```

Comment:
Remove space before brightly (there is already on on the previous line; that is the correct place for the space).